src/gui/usbipd_gui.rs

```diff
@@ -154,9 +141,102 @@ impl UsbipdGui {
         self.window.set_visible(true);
     }
 
-    fn show_tray_menu(&self) {
+    fn show_menu_tray(self: &Rc<UsbipdGui>) {
+        if let Some(handler) = self.menu_tray_event_handler.take() {
+            nwg::unbind_event_handler(&handler);
+        }
+
+        let mut menu_tray = nwg::Menu::default();
+        nwg::Menu::builder()
+            .popup(true)
+            .parent(self.window.handle)
+            .build(&mut menu_tray)
+            .unwrap();
+
+        let devices = list_devices()
+            .into_iter()
+            .filter(|d| d.is_connected())
+            .collect::<Vec<_>>();
+
+        let mut menu_items: Vec<(nwg::MenuItem, UsbDevice)> = Vec::with_capacity(devices.len());
+        for device in devices {
+            let device_name = device.description.as_deref();
+            let vid_pid = device.vid_pid();
+            let description = device_name.map(|s| s.to_string()).unwrap_or(
+                vid_pid
+                    .clone()
+                    .unwrap_or_else(|| "Unknown Device".to_string()),
+            );
+
+            if device.is_bound() {
+                let menu_item = self
+                    .new_menu_item(menu_tray.handle, &description, device.is_attached())
+                    .unwrap();
+
+                menu_items.push((menu_item, device));
+            }
+        }
+
+        self.new_menu_separator(menu_tray.handle).unwrap();
```

Comment:
Ideally this separator should only show when at least one device is displayed in the menu, otherwise it wouldn't look good. Alternatively, we can show a greyed-out (disabled) option saying something like "No bound devices" to hint the user that bound devices will be shown there. Let me know what you think.

Reply:
I like the greyed-out "No bound devices" - simpler implementation and self-documenting.